services/search/search_engine.py

```diff
@@ -94,20 +99,11 @@ def search_query(self, keywords, synonyms):
 
         score_per_article = {}
 
-        # keywords_json = {"keywords": keywords}
-        # synonyms_json = {"keywords": synonyms}
-        # article_keywords_frequency = requests.post(db_endpoint, json=keywords_json)
-        # article_synonyms_frequency = requests.post(db_endpoint, json=synonyms_json)
-
-        article_keywords_frequency = testing.test_constants.KEYWORDS_DB_MOCK_1
-        article_synonyms_frequency = testing.test_constants.SYNONYMS_DB_MOCK_1
+        keywords_json = {"keywords": keywords+synonyms}
+        article_keywords_frequency = requests.post(os.getenv('DB_ENDPOINT'), json=keywords_json)
```

Comment:
Are you planning to do any error handling for the `DB_ENDPOINT`??

Reply:
Talked with @jaimehisao and he will respond with errors in the same format as me